docs/cromwell_features/WorkflowCallback.md

```diff
@@ -0,0 +1,57 @@
+The workflow callback is a simple way to integrate Cromwell with an external system. When each workflow reaches a terminal
+state, Cromwell will POST a message to a provided URL (see below for schema of this message). Messages are sent for root
+workflows only, not subworkflows.
```

Comment:
One last thought - maybe include the phase "best effort" here to make sure it clear that Cromwell will try, but makes no guarantees about delivery